pkg/larker/fs/github/github.go

```diff
@@ -21,6 +22,12 @@ type GitHub struct {
 	reference string
 }
 
+type IsDir bool
```

Comment:
>Seems a bit too smart and if we'll want to add file size this will need to away.

See https://github.com/cirruslabs/cirrus-cli/pull/93/commits/7bfb0984f03210aadd480941ce0e13111adba375.